lib/ex_debug_toolbar/data/collection.ex

```diff
@@ -8,3 +8,9 @@ alias ExDebugToolbar.Data.Collection
 defimpl Collection, for: Map do
   def change(collection, map), do: Map.merge(collection, map)
 end
+
+defimpl Collection, for: List do
+  def change(collection, item) do 
+    [item | Enum.reverse(collection)] |> Enum.reverse
```

Comment:
this double reversing feels wrong. 
I had the same problem with the timeline. Do you know how I solved it? I didn't =)
i think this should be solved when exporting the logs to the ui. another method like `finalize(collection)` that is implemented the same way as the change/2 is